### ISSUE TITLE
Redis secret is not populated

### ISSUE THREAD

**Issue by staff0rd** [dotnet/tye#309]
### Describe the bug
I get 500 when the redis example attempts to connect to redis once deployed to k8s.

### To Reproduce

1. Follow [redis example here](https://github.com/dotnet/tye/blob/master/docs/redis.md).
1. `kubectl port-forward svc/backend 5000:80`
1. `curl -v http://localhost:5000/weatherforecast`

Expected: 200
Actual: 500

### Logs
```
fail: Microsoft.AspNetCore.Server.Kestrel[13]
      Connection id "0HLUQP4KKQCLD", Request id "0HLUQP4KKQCLD:00000001": An unhandled exception was thrown by the application.
StackExchange.Redis.RedisConnectionException: It was not possible to connect to the redis server(s). UnableToConnect on ::6379/Interactive, Initializing/NotStarted, last: NONE, origin: BeginConnectAsync, outstanding: 0, last-read: 0s ago, last-write: 0s ago, keep-alive: 60s, state: Connecting, mgr: 10 of 10 available, last-heartbeat: never, global: 202s ago, v: 2.0.593.37019
   at StackExchange.Redis.ConnectionMultiplexer.ConnectImplAsync(Object configuration, TextWriter log) in C:\projects\stackexchange-redis\src\StackExchange.Redis\ConnectionMultiplexer.cs:line 825
   at Microsoft.Extensions.Caching.StackExchangeRedis.RedisCache.ConnectAsync(CancellationToken token)
   at Microsoft.Extensions.Caching.StackExchangeRedis.RedisCache.GetAndRefreshAsync(String key, Boolean getData, CancellationToken token)
   at Microsoft.Extensions.Caching.StackExchangeRedis.RedisCache.GetAsync(String key, CancellationToken token)
   at Microsoft.Extensions.Caching.Distributed.DistributedCacheExtensions.GetStringAsync(IDistributedCache cache, String key, CancellationToken token)
   at backend.Controllers.WeatherForecastController.Get(IDistributedCache cache, IConfiguration config) in c:\git\microservice\backend\Controllers\WeatherForecastController.cs:line 44
   at lambda_method(Closure , Object )
   at Microsoft.Extensions.Internal.ObjectMethodExecutorAwaitable.Awaiter.GetResult()
   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.AwaitableObjectResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeActionMethodAsync>g__Awaited|12_0(ControllerActionInvoker invoker, ValueTask`1 actionResultValueTask)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeNextActionFilterAsync>g__Awaited|10_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeInnerFilterAsync>g__Awaited|13_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeFilterPipelineAsync>g__Awaited|19_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Server.Kestrel.Core.Internal.Http.HttpProtocol.ProcessRequests[TContext](IHttpApplication`1 application)
```

### Further technical details
Version
```
λ tye --version
0.1.0-alpha.20202.7+f0bb65979475c139096cc24b0fafaebd524dae4f
```

Appears the secret is imported into the container;
```
λ kubectl exec backend-dc7cdc744-dvsc7 -- cat /var/tye/bindings/redis-redis/CONNECTIONSTRING__REDIS
redis:6379
```

**Comment by rynowak** [comment]
Thanks for the report!

This should be fixed by #299 - I'll leave this issue open until I can verify that a package in the wild exists that solves this. I want to apologize for leaving this tutorial in a half-done state. We made a new package and put this functionality inside, but didn't update the docs to cover where to get the package, and also the package had a bug 👎 

If you want a workaround for now, but the code from https://github.com/dotnet/tye/pull/299/files#diff-d2111fc02775913311ddbcf1ff4f3e28R26 should do

**Comment by rynowak** [comment]
Tested this, and it's fixed with the latest CI build. Thanks again.

Note that the instructions have been updated to include some details that were missed.
